<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SysPraModel</class>
 <widget class="QWidget" name="SysPraModel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>906</width>
    <height>510</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>401</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>参数设置</string>
   </property>
   <widget class="QPushButton" name="pushButtonWritePra">
    <property name="geometry">
     <rect>
      <x>246</x>
      <y>98</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>写入参数</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCOT">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>81</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_3">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>81</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>COT：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_27">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>81</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_28">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>109</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCUT">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>109</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_4">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>109</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CUT：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDOT">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>137</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_29">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>137</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_5">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>137</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DOT：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_11">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>165</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DUT：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_49">
    <property name="geometry">
     <rect>
      <x>103</x>
      <y>165</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDUT">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>165</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-50</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>50</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCOV">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>25</y>
      <width>52</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2000</number>
    </property>
    <property name="maximum">
     <number>4500</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>3500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_25">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>25</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mV</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>25</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>COV：</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_2">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>53</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>CUV：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCUV">
    <property name="geometry">
     <rect>
      <x>51</x>
      <y>53</y>
      <width>52</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>2000</number>
    </property>
    <property name="maximum">
     <number>4500</number>
    </property>
    <property name="singleStep">
     <number>100</number>
    </property>
    <property name="value">
     <number>3500</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_26">
    <property name="geometry">
     <rect>
      <x>109</x>
      <y>53</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mV</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_50">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>25</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCOC">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>25</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>20</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_12">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>25</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>COC：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDOC">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>53</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>500</number>
    </property>
    <property name="value">
     <number>200</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_13">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>53</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DOC：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_51">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>53</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_52">
    <property name="geometry">
     <rect>
      <x>225</x>
      <y>81</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>mV</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDIV">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>81</y>
      <width>52</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
    <property name="singleStep">
     <number>10</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_14">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>81</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DIV：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxDLT">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>109</y>
      <width>40</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>50</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_15">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>109</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>DLT：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_53">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>109</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>℃</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_54">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>137</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_16">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>137</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>POV：</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxPOV">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>137</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>85</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxCUV_3">
    <property name="geometry">
     <rect>
      <x>167</x>
      <y>165</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>60</number>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_17">
    <property name="geometry">
     <rect>
      <x>131</x>
      <y>165</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>PUV：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_55">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>165</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>V</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxISO">
    <property name="geometry">
     <rect>
      <x>283</x>
      <y>25</y>
      <width>46</width>
      <height>20</height>
     </rect>
    </property>
    <property name="minimum">
     <number>0</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_56">
    <property name="geometry">
     <rect>
      <x>335</x>
      <y>25</y>
      <width>16</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ω</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelSetCOV_18">
    <property name="geometry">
     <rect>
      <x>247</x>
      <y>25</y>
      <width>30</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ISO：</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxPackInfo">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>0</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>电池包信息</string>
   </property>
   <widget class="QWidget" name="layoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_21">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>串数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelSeriesNum">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_22">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>并数：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelParallelNum">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="1,2">
       <item>
        <widget class="QLabel" name="label_23">
         <property name="font">
          <font>
           <family>Agency FB</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>容量：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelCapacity">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_24">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSysPra">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>291</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>系统参数</string>
   </property>
   <widget class="QTableView" name="tableViewSysPra">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>271</width>
      <height>461</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
